shared/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -131,7 +131,10 @@ private String colorSchemeForCurrentConfiguration() {
   @TargetApi(Build.VERSION_CODES.KITKAT)
   protected void onCreate(Bundle savedInstanceState) {
     ReactInstanceManager instanceManager = this.getReactInstanceManager();
-    instanceManager.createReactContextInBackground();
+    if (!instanceManager.hasStartedCreatingInitialContext()) {
```

Comment:
I thought `createReactContextInBg` was a no-op if called twice, but apparently not? RN got mad at me, so I added this check